platform/ios/docs/pod-README.md

```diff
@@ -33,7 +33,17 @@ bash "${BUILT_PRODUCTS_DIR}/${FRAMEWORKS_FOLDER_PATH}/Mapbox.framework/strip-fra
 
 If your application targets iOS 7.x, you’ll need to install the static framework instead:
 
-1. Open the project editor and select your application target. Drag `Mapbox.framework` from the `static` folder into the “Embedded Binaries” section of the General tab. (Don’t drag it into the “Linked Frameworks and Libraries” section; Xcode will add it there automatically.) In the sheet that appears, make sure “Copy items if needed” is checked, then click Finish.
+1. Build from source manually per above.
```

Comment:
@friedbunny @1ec5 

Sorry, this was copy pasta from the INSTALL.md file. I will remove this line. I see that there is language referencing the releases page above in the `## Installation` section. Is that good enough or do we need to be more specific here and in the DYNAMIC template just above?